### DISCUSSION TITLE
Vue port

### DISCUSSION THREAD

**Discussion by pavitra-infocusp** [tremorlabs/tremor-npm#1116] [General]
Tremor components looks really nice. I would love to use these in my Vue projects. Do you intend to provide a Vue version anytime soon?

Otherwise, I am wondering if I may be able to port these components to Vue. According to the [License](https://github.com/tremorlabs/tremor/blob/3c67e41b07e8c3f20efc7395d84bfb8cde292119/License), derivative works/ modification is allowed. If desirable, I can also help to contribute the Vue port here.

**Comment by severinlandolt** [comment]
Hi! We don't have any plans to add Vue support. Feel free to create a port :)

**Reply by pavitra-infocusp** [reply]
Thanks for the approval!
